aip-console-jenkins/README.md

```diff
@@ -32,35 +33,35 @@ Click on Upload and Jenkins should install the plugin.
 
 ### Configuration
 
-To configure the AIP Console plugin, go to the "Configure System" page in the "Manage Jenkins" page.
+To configure the CAST Imaging jenkins plugin, go to the "Configure System" page in the "Manage Jenkins" page.
 
 <img src="doc/images/configure_system_menu_item.png" alt="global configuration page location" style="zoom:50%;" />
 
-On this page, go to the **AIP Console global configuration** section and enter the following required elements :
+On this page, go to the **Imaging Console global configuration** section and enter the following required elements :
 
-* *AIP Console URL* : The URL to access the AIP Console server. Make sure that this address is accessible from the
+* *Imaging Console URL* : The URL to access the CAST Imaging server. Make sure that this address is accessible from the
   Jenkins Server and the Jenkins Node that might run the plugin as well.
-* *API Key* : The API key that will be used to authenticate to AIP Console.
-* *Timeout* : A global timeout setting in seconds before calls to AIP Console will be considered in error.
-* *verbose* : When checked, the AIP Console logs will also be displayed on the build's log. Uncheck this to prevent displaying the entirety of the AIP Console job execution to be displayed during the build.
+* *API Key* : The API key that will be used to authenticate to CAST Imaging.
+* *Timeout* : A global timeout setting in seconds before calls to CAST Imaging will be considered in error.
+* *verbose* : When checked, the CAST Imaging logs will also be displayed on the build's log. Uncheck this to prevent displaying the entirety of the CAST Imaging job execution to be displayed during the build.
 
 ![Global configuration page](./doc/images/global_config_page.png)
 
-### Using different AIP Console URL and API Key
+### Using different Imaging Console URL and API Key
 
-Should you require to target a specific AIP Node instance that differs from the global configuration in Jenkins, you can specify the URL and an API Key by opening the "Advanced Settings" options. Note that you will have to manually set them for each build steps that require targeting a specific AIP Console Instance.
+Should you require to target a specific AIP Node instance that differs from the global configuration in Jenkins, you can specify the URL and an API Key by opening the "Advanced Settings" options. Note that you will have to manually set them for each build steps that require targeting a specific CAST Imaging Instance.
 
-![Step specific AIP Console URL and API Key configuration](./doc/images/custom_URL_ApiKey.png)
+![Step specific Imaging Console URL and API Key configuration](./doc/images/custom_URL_ApiKey.png)
```

Comment:
```suggestion
![Step specific CAST Imaging Console URL and API Key configuration](./doc/images/custom_URL_ApiKey.png)
```
I think the product name is "CAST Imaging Console" so we should probably make sure it is coherent everywhere